src/test/java/org/opensearch/neuralsearch/query/HybridQueryIT.java

```diff
@@ -578,6 +579,54 @@ public void testRequestCache_whenMultipleShardsQueryReturnResults_thenSuccessful
         }
     }
 
+    @SneakyThrows
+    public void testWrappedQueryWithFilter_whenIndexAliasHasFilterAndIndexWithNestedFields_thenSuccess() {
```

Comment:
can we have an integ test where we just alias filters.

Reply:
sure, added test specifically for that scenario